src/main/java/application/rest/v1/ResourcesEndpoint.java

```diff
@@ -0,0 +1,1798 @@
+
+    //Ingress constants
+    private static final String INGRESS_KIND = "Ingress";
+    private static final String INGRESSE_PROPERTY_NAME = "Ingresses";
+    private static final String INGRESS_GROUP = "config.openshift.io";
+    private static final String INGRESS_PLURAL = "ingresses";
+
+    //IngressController constants 
+    private static final String INGRESS_CONTROLLER_KIND = "IngressController";
+    private static final String INGRESS_CONTROLLER_PROPERTY_NAME = "IngressControllers";
+    private static final String INGRESS_CONTROLLER_PLURAL = "ingresscontrollers";
+
+    //Job constants
+    private static final String JOB_KIND = "Job";
+    private static final String JOB_PROPERTY_NAME = "Jobs";
+    private static final String JOB_GROUP = "batch";
+    private static final String JOB_PLURAL = "jobs";
+
+    //Kappnav constants
+    private static final String KAPPNAV_KIND = "Kappnav";
+    private static final String KAPPNAV_PROPERTY_NAME = "Kappnavs";
+    private static final String KAPPNAV_CR_GROUP = "kappnav.operator.kappnav.io";
+    private static final String KAPPNAV_PLURAL = "kappnavs";
+
+    //KindActionMapping constants
+    private static final String KAM_KIND = "KindActionMapping";
+    private static final String KAM_PROPERTY_NAME = "KindActionMappings";
+    private static final String KAM_GROUP = "actions.kappnav.io";
+    private static final String KAM_PLURAL = "kindactionmappings";
+
+    //KubeAPIServer constants
+    private static final String KUBE_API_SERVER_KIND = "KubeAPIServer";
+    private static final String KUBE_API_SERVER_PROPERTY_NAME = "KubeAPIServers";
+    private static final String KUBE_API_SERVER_PLURAL = "kubeapiservers";
+
+    //KubeControllerManagerconstants
+    private static final String KUBE_CONTROLLER_MANAGER_KIND = "KubeControllerManager";
+    private static final String KUBE_CONTROLLER_MANAGER_PROPERTY_NAME = "KubeControllerManagers";
+    private static final String KUBE_CONTROLLER_MANAGER_PLURAL = "kubecontrollermanagers";
+
+    //KubeScheduler constants
+    private static final String KUBE_SCHEDULER_KIND = "KubeScheduler";
+    private static final String KUBE_SCHEDULER_PROPERTY_NAME = "KubeSchedulers";
+    private static final String KUBE_SCHEDULER_PLURAL = "kubeschedulers";
+
+    //KubeletConfig constants
+    private static final String KUBELET_CONFIG_KIND = "KubeletConfig";
+    private static final String KUBELET_CONFIG_PROPERTY_NAME = "KubeletConfigs";
+    private static final String KUBELET_CONFIG_GROUP = "machineconfiguration.openshift.io";
+    private static final String KUBELET_CONFIG_PLURAL = "kubeletconfigs";
+   
+    //Lease constants 
+    private static final String LEASE_KIND = "Lease";
+    private static final String LEASE_PROPERTY_NAME = "Leases";
+    private static final String LEASE_GROUP = "coordination.k8s.io";
+    private static final String LEASE_PLURAL = "leases";
+
+    //Liberty-App constants
```

Comment:
Must not refer to Liberty and WAS kinds in this file.  If they must be referenced, it must be done using the "extensions" technique, used elsewhere - e.g. this line (approximately line 137) in ComponentInfoRegistry.java:

```
 KAppNavExtension.init();
```

Reply:
ok, I think that I can add a new ResourcesExtensionEndpoint in prism site to handle twas and liberty kinds of resources.